PPV12/PPV12/AppDelegate.m

```diff
@@ -18,13 +21,30 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    // Override point for customization after application launch.
+    // Venmo app info
+    [Venmo startWithAppId:@"2431" secret:@"3XA9f8Qc99HxSR9j8mdLtv7vabS8n5yy" name:@"PayPal12"];
```

Comment:
Do we want this to be hardcoded in the source file? Anyone can abuse this? :no_bell: